src/QuadForm/Lattices.jl

```diff
@@ -966,6 +966,45 @@ function restrict_scalars(L::AbsLat)
   return ZLat(Vabs, Mabs)
 end
 
+#=
+    _restrict_scalars_with_map(L) -> 
+      Tuple{ZLat, Hecke.VecSpaceRes{Hecke.NfRel{nf_elem}, Hecke.NfRelElem{nf_elem}}}
+
+Takes a lattice $L$ and returns a $\mathbb{Z}$-Lattice and the restriction map $f$.
+=#
```

Comment:
```suggestion
#=
Takes a lattice $L$ and return a $\mathbb{Z}$-Lattice and the restriction map $f$.
=#
```
Internal comments: put only the necessary. To be applied elsewhere